package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@fluentui/fluentui-charting-contrib",
+    "version": "1.0",
+    "description": "Experimental React charting Contribution environment for building experiences for React charting library.",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/fluentui-charting-contrib"
+    },
+    "license": "MIT",
+    "scripts": {
+        "test": "jest"
+    },
+    "devDependencies": {
+        "jest": "29.7.0",
```

Comment:
import just react-charting, rewire, RTL, jest etc

Reply:
yes, rest are required as they are part of the SparklineBase.js.

Reply:
done